stripper/ze_diverted_destiny_c2f3.cfg

```diff
@@ -0,0 +1,381 @@
+;stripper by koen (STEAM_1:1:114921174)
+
+;remove broken trigger_hurt for solar flare
+filter:
+{
+	"classname" "trigger_hurt"
+	"targetname" "solarflare_item_h"
+	"hammerid" "328010"
+}
+
+;adjust solar flare damage to prevent repeat killer
+modify:
+{
+	match:
+	{
+		"classname" "trigger_hurt"
+		"targetname" "solarflare_item_h"
+		"hammerid" "328019"
+	}
+	replace:
+	{
+		"damage" "2500"
```

Comment:
This is a fair bit lower than we discussed, and probably has more balance implications?